### DISCUSSION TITLE
💆‍♂️Sudo issue on Android termux 🤧

### DISCUSSION THREAD

**Discussion by djr56998** [termux/termux-app#2375] [Q&A] [answered]
I am trying to run sudo command for kali Linux but the syntax returns as👇, how do I fixed this this? And what does this mean?
└─$ sudo
sudo: /etc/sudo.conf is owned by uid 100000, should be 0
sudo: /usr/bin/sudo must be owned by uid 0 and have the setuid bit set

**Comment by nathaneltitane** [comment, marked as answer]
definitely some kind of configuration errror: happens at random for me - i have a proot setup and this happens once every 10 or 11 installs oddly
